src/system-health/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+filterwarnings =
+    ignore::DeprecationWarning
```

Comment:
I think we can do away with this warning.

Also, I suggest adding options to print out test coverage reports. Please use the following as an example: https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-py-common/pytest.ini

Reply:
Fixed